src/pages/workspace/expensifyCard/EmptyCardView.tsx

```diff
@@ -39,6 +39,7 @@ function EmptyCardView() {
                     title={translate('workspace.expensifyCard.issueAndManageCards')}
                     subtitle={translate('workspace.expensifyCard.getStartedIssuing')}
                     emptyStateForegroundStyles={isSmallScreenWidth && {justifyContent: 'flex-start'}}
+                    minModalHeight={500}
```

Comment:
Can we please move this value (and the default `400` above) to `src/variables`, since they're more akin to styling values?